### PR TITLE
AOM-23: Write test for manage-addon page

### PR DESCRIPTION
**Pull request by jomadoye**
# JIRA TICKET NAME:
AOM-22: Write test for manage-addon page](https://issues.openmrs.org/browse/AOM-23)

## SUMMARY:
This PR adds tests for the manage-addon page

### CODE REVIEW

**Review by wanjikum — COMMENTED** [review]
Good work Jedidah!

**Review by paulupendo — COMMENTED** [review]
Good work Jed.

File: tests/homepage/ManageAddon.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import ManageApps from '../../app/js/components/homePage/manageApps/manageApps';
+import BreadCrumbComponent from '../../app/js/components/breadCrumb/breadCrumbComponent';
+import AddAddon from '../../app/js/components/homePage/manageApps/AddAddon';
```
Comment (kingisaac95) [inline tests/homepage/ManageAddon.test.js:6]: Any reason you have this import (`AddAddon`) here since it wasn't used anywhere?

Comment (dkayiwa) [reply tests/homepage/ManageAddon.test.js:6]: Good question @kingisaac95 
@andela-jomadoye can you address it in a followup pull request?

File: tests/homepage/ManageAddon.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        expect(renderedComponent.find("th").getNodes()[1]
+            .props.children).to.equal('Name');
+        expect(renderedComponent.find("th").getNodes()[2]
+            .props.children).to.equal('Developer');
+        expect(renderedComponent.find("th").getNodes()[3]
+            .props.children).to.equal('Version');
+        expect(renderedComponent.find("th").getNodes()[4]
+            .props.children).to.equal('Delete');
+    });
+
```
Comment (kingisaac95) [inline tests/homepage/ManageAddon.test.js:46]: Good job testing the table head elements @andela-jomadoye

Comment (andela-jomadoye) [reply tests/homepage/ManageAddon.test.js]: Thanks @kingisaac95, @paulupendo @wanjikum @dkayiwa

**Comment by jomadoye** [comment]
Thanks @kingisaac95, @paulupendo @wanjikum @dkayiwa

**Pull request merged**